api/mesh/v1alpha1/mesh/rest.yaml

```diff
@@ -8,6 +8,7 @@ info:
 paths:
   /meshes/{name}:
     get:
+      operationId: get-Mesh
```

Comment:
nit: mixing dashes and capitalised letters looks a bit unusual, is there a style guide? can we use `getMesh` or `get-mesh` instead?

Reply:
that would have to be an additional field here https://github.com/kumahq/kuma/blob/3aef2b26560e7634f0688065f1be630863c0eaa3/pkg/plugins/policies/meshaccesslog/api/v1alpha1/zz_generated.resource.go#L136 or adding a template function like "lowercase" or I can use shortname https://github.com/kumahq/kuma/blob/3aef2b26560e7634f0688065f1be630863c0eaa3/pkg/plugins/policies/meshaccesslog/api/v1alpha1/zz_generated.resource.go#L156

Reply:
I think we already [guarantee](https://github.com/kumahq/kuma/blob/3aef2b26560e7634f0688065f1be630863c0eaa3/pkg/core/resources/registry/registry.go#L100) that `ShortName` is unique, so I'd just use it

Reply:
used "shortName".